src/meta/src/hummock/compaction_group/mod.rs

```diff
@@ -40,6 +42,8 @@ impl CompactionGroup {
             member_table_ids: Default::default(),
             compaction_config,
             table_id_to_options: HashMap::default(),
+            father_group_id: StaticCompactionGroupId::NewCompactionGroup as CompactionGroupId,
```

Comment:
What does `NewCompactionGroup` mean ?

Reply:
> What does `NewCompactionGroup` mean ?

Here it means that this group does not have a `father_group`.
Anyway, `father_group` is a temporary design for split. We need a more complete design when we want group merging.